packages/runtime/src/next/preview-mode.tsx

```diff
@@ -12,12 +12,12 @@ const makeswiftPreviewDataSchema = z.object({
 })
 export type MakeswiftPreviewData = z.infer<typeof makeswiftPreviewDataSchema>
 
-export function getMakeswiftSiteVersion(previewData: PreviewData): MakeswiftSiteVersion | null {
+export function getMakeswiftSiteVersion(previewData: PreviewData): MakeswiftSiteVersion {
   const result = makeswiftPreviewDataSchema.safeParse(previewData)
 
   if (result.success) return result.data.unstable_siteVersion
 
-  return null
+  return MakeswiftSiteVersion.Live
```

Comment:
I'm not sure I'd want to place this here—it combines picking a default site version with extracting one from the preview data. The purpose of this function was extracting it from the preview data and defaulting it like this here makes it impossible to know, given the returned value of this function, whether it was explicitly set to live or if it defaulted to that

Reply:
Agree w that